mix.exs

```diff
@@ -37,13 +37,23 @@ defmodule Membrane.RTSP.Plugin.Mixfile do
   defp deps do
     [
       {:membrane_core, "~> 1.1"},
-      {:membrane_rtsp, "~> 0.7.1"},
+      {:membrane_rtsp,
+       github: "membraneframework/membrane_rtsp",
+       branch: "allow-repeating-requests",
+       override: true},
+      # {:membrane_rtsp, "~> 0.7.1"},
```

Comment:
Should we wait for the new releases of this deps ?

Reply:
yes